docs/tutorial/04-pipeline.ipynb

```diff
@@ -55,7 +55,8 @@
     "* `mlrun.projects.get_or_create_project()` &mdash; Load a project from the MLRun DB if it exists, or from a specified \n",
     "  context/archive. \n",
     "\n",
-    "Projects refer to a `context` directory that holds all the project code and configuration. The `context` dir is \n",
+    "Projects refer to a `context` directory that holds all the project code and configuration. Its default value is \"./\". \n",
+    "The `context` dir is \n",
```

Comment:
I think it is better to explain the "./" - which is the directory the mlrun client runs from.